### PR TITLE
signal constants simplified and some drying in goobj/read parseObject

### PR DESCRIPTION
**Pull request by vural**

### CODE REVIEW

**Comment by vural** [comment]
I signed it!

File: signal/signal.go [hunk 1]
```diff
@@ -7,23 +7,23 @@ package signal
 
 const (
 	// StackTrace represents a command to print stack trace.
-	StackTrace = byte(0x1)
+	StackTrace = byte((iota + 1))
```
Comment (shurcooL) [inline signal/signal.go]: 2 sets of round brackets can be simplified to just one.

File: internal/goobj/read.go [hunk 2]
```diff
@@ -679,8 +679,6 @@ func (r *objReader) parseObject(prefix []byte) error {
 			f.FuncData = make([]FuncData, r.readInt())
 			for i := range f.FuncData {
 				f.FuncData[i].Sym = r.readSymID()
-			}
-			for i := range f.FuncData {
 				f.FuncData[i].Offset = int64(r.readInt()) // TODO
```
Comment (shurcooL) [inline internal/goobj/read.go]: I'm not very familiar with this project, but doesn't this change the behavior? The order was previously like `r.readSymID(), r.readSymID(), r.readSymID(), r.readInt(), r.readInt(), r.readInt()`, but now it'll be `r.readSymID(), r.readInt(), r.readSymID(), r.readInt(), r.readSymID(), r.readInt()`.

File: signal/signal.go (hunk 1 above)
Comment (vural) [inline signal/signal.go]: I think it's better https://github.com/google/gops/pull/21/commits/7d9086f8c133c523692146f42b99ba8cb7ef72e6

File: internal/goobj/read.go (hunk 2 above)
Comment (vural) [inline internal/goobj/read.go]: Yes you are right, changing your behavior.

Comment (rakyll) [inline internal/goobj/read.go]: Please, do not modify the internal packages. They are vendored from the Go project.

File: signal/signal.go [hunk 3]
```diff
@@ -7,23 +7,23 @@ package signal
 
 const (
 	// StackTrace represents a command to print stack trace.
-	StackTrace = byte(0x1)
```
Comment (rakyll) [inline signal/signal.go]: Please keep these as they are. I don't want the reordering here break the clients compiled with a different version in an incompatible way.

**Comment by vural** [comment]
Thank you, I closed this PR.

**Pull request closed without merging**